pkg/pillar/scripts/device-steps.sh

```diff
@@ -172,8 +218,9 @@ fi
 
 # Run upgradeconverter
 echo "$(date -Ins -u) device-steps: Starting upgradeconverter"
-status=$($BINDIR/upgradeconverter)
-echo "$(date -Ins -u) device-steps: upgradeconverter Completed. Status: $status"
+$BINDIR/upgradeconverter &
+wait_for_done upgradeconverter 600
```

Comment:
This should be infinite as well to match current behavior

Reply:
Done.